### PR TITLE
Improve the infrastructure of the project

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Moved node-sass to DartSass
- Moved lint-scss to stylelist
- Created a `npm run serve` endpoint to running the server

## QA
- Run `npm run clean`
- Run `npm install`
- Run `npm run buiild`
- Run `npm run serve`
- Go to http://0.0.0.0:8301/ and see it works as before.

### CODE REVIEW

**Comment by pmahnke** [comment]
@anthonydillon this didn't work for me.

I had to do `npm run build` instead of just `npm build` and then `npm serve` didn't work, so I tried `npm run serve`, but that failed as well.

```
 npm run serve

> @canonical/cookie-policy@3.0.6 serve /home/peter/src/cookie-policy
> python -m SimpleHTTPServer 8301 && echo 'Demo is running on http://0.0.0.0:8301'

sh: 1: python: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @canonical/cookie-policy@3.0.6 serve: `python -m SimpleHTTPServer 8301 && echo 'Demo is running on http://0.0.0.0:8301'`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @canonical/cookie-policy@3.0.6 serve script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

**Comment by anthonydillon** [comment]
Ah I thought python would be a standard package in Ubuntu. Thanks, will think of something else.

**Comment by pmahnke** [comment]
I have python, but it is called `python2.7` I have to run `python2.7 -m SimpleHTTPServer`. But perhaps I have messed up my aliases? I have nothing at `python`.  I also have `python2             python2.7 python3             python3.6           python3.8`

**Comment by anthonydillon** [comment]
@pmahnke updated to use python3 and updated the Readme.

**Comment by pmahnke** [comment]
@anthonydillon ok, now it runs, but the Japanese link isn't there

image

where Chinese is fine

image

**Comment by anthonydillon** [comment]
@pmahnke Updated

File: ja/index.html [hunk 1]
```diff
@@ -32,9 +32,12 @@
 
   <body>
     <h1>The Japanese cookie manager</h1>
-    <p>これは、CanonicalsWebサイトのCookieマネージャープロジェクトのデモです。</a>
-    <p><a href="/"> English version </a></p>
-    <p><a href="/zh"> Chinese version </a></p>
+    <p>
+      これは、CanonicalsWebサイトのCookieマネージャープロジェクトのデモです。
+    </p>
+    <p><a href="" class="js-revoke-cookie-manager">撤消cookie管理器</a></p>
```
Comment (pmahnke) [inline ja/index.html]: I _think_ this text is Chinese, not Japanese?

**Comment by anthonydillon** [comment]
@pmahnke oh sorry, working in meetings :facepalm: Updated.

**Review by pmahnke — APPROVED** [review]
+1 - this will make it easier for everyone to review.

**Pull request merged**